CHANGELOG.md

```diff
@@ -1,4 +1,39 @@
 ## 4.80.0 (Unreleased)
+DEPRECATIONS:
+* dataplex: deprecated the following `google_dataplex_datascan` fields: `dataProfileResult` and `dataQualityResult` ([#15528](https://github.com/hashicorp/terraform-provider-google/pull/15528))
+* firebase: deprecated `google_firebase_project_location` in favor of `google_firebase_storage_bucket` and `google_firestore_database` ([#15526](https://github.com/hashicorp/terraform-provider-google/pull/15526))
+
+FEATURES:
+* **New Data Source:** `google_sql_database_instance_latest_recovery_time` ([#15551](https://github.com/hashicorp/terraform-provider-google/pull/15551))
+* **New Resource:** `google_certificate_manager_trust_config` ([#15562](https://github.com/hashicorp/terraform-provider-google/pull/15562))
+* **New Resource:** `google_compute_region_security_policy_rule` ([#15523](https://github.com/hashicorp/terraform-provider-google/pull/15523))
+* **New Resource:** `google_gke_hub_membership_rbac_role_binding` ([#15544](https://github.com/hashicorp/terraform-provider-google/pull/15544))
```

Comment:
```suggestion
```

hotfixing removal of this resource

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/8776

Reply:
https://github.com/hashicorp/terraform-provider-google/commit/feb52d5adb7591bff1b7932471fe82e2feb4d2f7